H13116004/Praktikum_#1/Tugas05.java

```diff
@@ -10,7 +10,7 @@ public static void main(String[] args) {
         int b = input.nextInt();
 
         boolean c = (a%b == 0 && b%a == 0);
-        c = a != b;
+        c = a != b || a == b;
```

Comment:
Baris ini akan selalu menghasilkan nilai true

Reply:
ok. i think i solve this. one sec